index.js

```diff
@@ -549,10 +549,9 @@ Lexer.prototype = {
    */
 
   path: function() {
-    var tok = this.scanEndOfLine(/^ +([^\n]+)/, 'path');
-    // the .trim() is necessary since we want to avoid `  ` from being
-    // matched, as the second space satisfies `[^\n]+`
+    var tok = this.scanEndOfLine(/^ ([^\n]+)/, 'path');
     if (tok && tok.val.trim()) {
+      tok.val = tok.val.trim();
```

Comment:
We are now running trim twice - how about adding the tok.val = assignment into the if statement?
```js
if (tok && "" !== (tok.val = tok.val.trim()))
```